src/www/ui/core-schema.dat

```diff
@@ -1006,6 +1006,22 @@
   $Schema["TABLE"]["license_ref_bulk"]["lrb_pk"]["DESC"] = "COMMENT ON COLUMN \"license_ref_bulk\".\"lrb_pk\" IS 'Primary Key'";
   $Schema["TABLE"]["license_ref_bulk"]["lrb_pk"]["ADD"] = "ALTER TABLE \"license_ref_bulk\" ADD COLUMN \"lrb_pk\" int8 DEFAULT nextval('license_ref_bulk_lrb_pk_seq'::regclass)";
   $Schema["TABLE"]["license_ref_bulk"]["lrb_pk"]["ALTER"] = "ALTER TABLE \"license_ref_bulk\" ALTER COLUMN \"lrb_pk\" SET NOT NULL, ALTER COLUMN \"lrb_pk\" SET DEFAULT nextval('license_ref_bulk_lrb_pk_seq'::regclass)";
+  
+  $Schema["TABLE"]["license_candidate"]["rf_creationdate"]["DESC"] = "COMMENT ON COLUMN \"license_candidate\".\"rf_creationdate\" IS 'group seeing this candidate'";
+  $Schema["TABLE"]["license_candidate"]["rf_creationdate"]["ADD"] = "ALTER TABLE \"license_candidate\" ADD COLUMN \"rf_creationdate\" timestamptz DEFAULT now()";
+  $Schema["TABLE"]["license_candidate"]["rf_creationdate"]["ALTER"] = "ALTER TABLE \"license_candidate\" ALTER COLUMN \"rf_creationdate\" DROP NOT NULL, ALTER COLUMN \"rf_creationdate\" SET DEFAULT now()";
+
+  $Schema["TABLE"]["license_candidate"]["rf_lastmodified"]["DESC"] = "COMMENT ON COLUMN \"license_candidate\".\"rf_lastmodified\" IS 'group seeing this candidate'";
+  $Schema["TABLE"]["license_candidate"]["rf_lastmodified"]["ADD"] = "ALTER TABLE \"license_candidate\" ADD COLUMN \"rf_lastmodified\" timestamptz DEFAULT now()";
+  $Schema["TABLE"]["license_candidate"]["rf_lastmodified"]["ALTER"] = "ALTER TABLE \"license_candidate\" ALTER COLUMN \"rf_lastmodified\" DROP NOT NULL, ALTER COLUMN \"rf_lastmodified\" SET DEFAULT now()";
+
+  $Schema["TABLE"]["license_candidate"]["rf_user_fk_created"]["DESC"] = "COMMENT ON COLUMN \"license_candidate\".\"rf_user_fk_created\" IS 'group seeing this candidate'";
+  $Schema["TABLE"]["license_candidate"]["rf_user_fk_created"]["ADD"] = "ALTER TABLE \"license_candidate\" ADD COLUMN \"rf_user_fk_created\" int4 DEFAULT 3";
+  $Schema["TABLE"]["license_candidate"]["rf_user_fk_created"]["ALTER"] = "ALTER TABLE \"license_candidate\" ALTER COLUMN \"rf_user_fk_created\" DROP NOT NULL";
+
+  $Schema["TABLE"]["license_candidate"]["rf_user_fk_modified"]["DESC"] = "COMMENT ON COLUMN \"license_candidate\".\"rf_user_fk_modified\" IS 'group seeing this candidate'";
```

Comment:
Please update the `DESC` (comment) for the added columns.

Reply:
yeah okay